src/components/container/ConnectEhr/ConnectEhr.tsx

```diff
@@ -95,26 +102,52 @@ export default function (props: ConnectEhrProps) {
 		return null;
 	}
 
+	if (props.hideWhenConnected && connected) {
+		return null;
+	}
+
+	let defaultTitle = language('connect-ehr-title-prefix') + language('connect-ehr-title-providers') + language('connect-ehr-title-divider') + language('connect-ehr-title-health-plans');
+	let title = props.title || defaultTitle;
+	let connectedText = props.connectedText || language("connect-ehr-text-connected").replace("@@PROJECT_NAME@@", projectName || "");
+	let notConnectedText = props.notConnectedText || language("connect-ehr-text").replace("@@PROJECT_NAME@@", projectName || "");
+	let text = (connected ? connectedText : notConnectedText);
+
+	let headerVariant = props.variant || "large";
+
+	let content = <>
+		<Title autosizeImage defaultMargin order={headerVariant == "large" ? 2 : 3} imageAlignment={headerVariant == "large" ? "top" : "left"} image={<img src={ConnectEHR} />}>{title}</Title>
+		<TextBlock>
+			{connected
+				? <>
+					<div className="connection-status">
+						{needsAttention
+							? <div className="warning">
+								<FontAwesomeSvgIcon icon={faTriangleExclamation} /> {language("connect-ehr-needs-attention")}
+							</div>
+							: <div className="success">
+								<FontAwesomeSvgIcon icon={faCheckCircle} /> {language("connect-ehr-connected")}
+							</div>
+						}
+					</div>
+					<div className="content">{text}</div>
+				</>
+				: <div className="content">{text}</div>
+			}
+		</TextBlock>
+	</>;
+
 	return (
 		<div ref={props.innerRef} className="mdhui-connect-ehr">
-			<Action bottomBorder={props.bottomBorder} title={language('connect-ehr-title-prefix') + language('connect-ehr-title-providers') + language('connect-ehr-title-divider') + language('connect-ehr-title-health-plans')} onClick={() => connectToEhr()}>
-				{connected
-					? <>
-						<div className="connection-status">
-							{needsAttention
-								? <div className="warning">
-									<FontAwesomeSvgIcon icon={faTriangleExclamation} /> {language("connect-ehr-needs-attention")}
-								</div>
-								: <div className="success">
-									<FontAwesomeSvgIcon icon={faCheckCircle} /> {language("connect-ehr-connected")}
-								</div>
-							}
-						</div>
-						<div className="content">{language("connect-ehr-text-connected").replace("@@PROJECT_NAME@@", projectName as any)}</div>
-					</>
-					: <div className="content">{language("connect-ehr-text").replace("@@PROJECT_NAME@@", projectName as any)}</div>
-				}
-			</Action>
+			{props.variant == "small" &&
+				<Action className="mdhui-connect-ehr-action" onClick={() => connectToEhr()}>
+					{content}
+				</Action>
+			}
+			{props.variant != "small" &&
+				<>{content}
+					<Button color={props.buttonColor} defaultMargin onClick={() => connectToEhr()}><FontAwesomeSvgIcon icon={faAddressCard} />&nbsp;&nbsp; {defaultTitle}</Button>
```

Comment:
This is just the call to action that opens the screen for provider search etc.  Wouldn't want provider search on the "home" screen for instance (since it's infinite scroll)